install-2x.html

```diff
@@ -244,7 +244,7 @@ <h2>3. Install Cypht on a YunoHost server</h2>
           YunoHost is an operating system that aims to simplify server administration as much as possible to democratize self-hosting while remaining reliable,
         secure, ethical and lightweight. It is a free software project owned exclusively by volunteers. Technically, it can be seen as a distribution based on
         Debian GNU/Linux and can be installed on many types of hardware.<br/>
-        To learn more about YunoHost, visit <a href="https://yunohost.org/fr/whatsyunohost">https://yunohost.org/fr/whatsyunohost</a>
+        To learn more about YunoHost, visit <a href="https://yunohost.org/en/overview/what_is_yunohost">https://yunohost.org/fr/whatsyunohost</a>
```

Comment:
part en, part fr

Reply:
Part en related to this: [check all links and make sure they are OK
Not broken, not a site in French, etc.](https://avan.tech/item105850)